content/docs/user-guide/index.md

```diff
@@ -1,12 +1,21 @@
 # User Guide
 
-Our guides describe the main DVC concepts and features comprehensively,
-explaining when and how to use them, as well as connections between them. These
-guides don't focus on specific scenarios, but have a general scope – like a user
-manual. Their topics range from more technical foundations, impacting more parts
-of DVC, to more advanced and specific things you can do. We also include a few
-guides related to contributing to
-[this open-source project](https://github.com/iterative/dvc).
+Our guides describe the major concepts and features of DVC comprehensively,
+explaining when and how to use them, as well as relationship between these. We
+don't focus on specific scenarios in this section, but rather on a general scope
+– think _User Manual_. The topics here range from more foundational, impacting
+more parts of DVC, to more technical and advanced things you can do. We also
+include a few misc. guides, for example related to
+[contributing to DVC](/doc/user-guide/contributing/core) itself.
+
+## Why DVC?
```

Comment:
probably can omit this

Reply:
I moved it to the Use Cases index instead now. It makes more sense in there, as it explains the general motivation for DVC to exist before going into a list of basic cases where it's useful. WDYT?